__test__/pages/search/searchResults.test.tsx

```diff
@@ -23,6 +24,20 @@ describe("Search Results page", () => {
       const cards = screen.getAllByRole("heading", { level: 3 })
       expect(cards).toHaveLength(50)
     })
+    it("renders the sort select field and updates the query string in the url on changes", async () => {
+      const query = "spaghetti"
+      await mockRouter.push(`/search?q=${query}`)
+      render(<SearchResults results={results} />)
+
+      const sortSelect = screen.getByLabelText("Sort by")
+      expect(sortSelect).toHaveValue("relevance")
+      await userEvent.selectOptions(sortSelect, "Title (A - Z)")
+      expect(sortSelect).toHaveValue("title_asc")
+
```

Comment:
your test implicitly tests that the page param is removed, but can it be its own expectation? just for documentation purposes.